kubernetes/gke-prow/prow/external-secrets.yaml

```diff
@@ -0,0 +1,104 @@
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: k8s-infra-cherrypick-robot-github-token
+spec:
+  data:
+  - remoteRef:
+      key: k8s-infra-cherrypick-robot-github-token
+    secretKey: token
+  secretStoreRef:
+    kind: ClusterSecretStore
+    name: kubernetes-public
+# ---
+# apiVersion: kubernetes-client.io/v1
+# kind: ExternalSecret
+# metadata:
+#   name: cluster-api-provider-digitalocean-quayio
+#   namespace: default
+# spec:
+#   backendType: gcpSecretsManager
+#   projectId: k8s-infra-prow-build-trusted
+#   data:
+#   - key: capdo-quayio-registry-secret
+#     name: config.json
+#     version: latest
```

Comment:
I think this is not used any longer, we should be able to delete it. Quay has ben used for CAPDO prior to donating the project to Kubernetes.